app/ui/faviconProgressbar.js

```diff
@@ -0,0 +1,48 @@
+const { platform } = require('../utils');
+const assets = require('../../common/assets');
+
+module.exports.updateFavicon = function(percentageString) {
+  if (platform() === 'web') {
+    const percentage = parseInt(percentageString.replace('%', ''));
```

Comment:
seems like you could clean this up a bit if you used `let` on this line and then maybe made the variable name a bit more clear by calling it `progress` instead of `percentage` since you're already using `percent` elsewhere

```
let progress = parseInt(percentageString.replace('%', ''));
progress = progress * .01; 
```

then on line 41 you can do `drawCircle(color, lineWidth, progress)